src/walletConnect/v1/reducer.ts

```diff
@@ -7,21 +8,29 @@ import { Actions, UserActions, WalletConnectActions } from 'src/walletConnect/v1
 
 export type PendingAction = { action: WalletConnectPayloadRequest; peerId: string }
 
+export enum WalletConnectDisplayedInfo {
+  None = 'none',
+  Basic = 'basic',
+  Full = 'full',
+}
```

Comment:
I propose we extract this to another file (since used by both WC and DAppKit) and give it a more generic name.
Maybe `DappsConnectInfo`? But should we have 2 types, one for controlling Connect and another for Signing? I guess this can be added later.
Can we also add a comment about this type and what each value does?

Reply:
haha i did go back and forth on this, i started adding this to the dapps slice and then decided it'd be better in the WC slice (but now i remember our conversation about wanting to keep supporting dappkit) so i've moved it back to the dapps slice now. i think that's a good home for this? 

i didn't make 2 variables though because the dappslist disclaimer is on both the connect and signing sheets. i added a comment to explain what each enum does though :)